admin/javascript/src/state/campaign/reducer.js

```diff
@@ -15,6 +15,9 @@ function campaignReducer(state = initialState, action) {
         view: action.payload.view,
       }));
 
+    case ACTION_TYPES.PUBLISH_CAMPAIGN_SUCCESS:
```

Comment:
I think this should unset `campaignId`? Also needs cases for `PUBLISH_CAMPAIGN_REQUEST` and `PUBLISH_CAMPAIGN_FAILURE` which should set / unset `campaignId`.